### ISSUE TITLE
threefold/actions_3script

### ISSUE THREAD

**Issue by despiegk** [crystaluniverse/crystallib#244]
> branch `development_web3gw`

These are tools to process incoming 3script actions and then talk to threefold/web3gw to get things executed

make some good examples in 3script in .md files

## dependencies

#243

**Comment by Omarabdul3ziz** [comment]
work pending on this https://github.com/freeflowuniverse/crystallib/issues/243

**Comment by ashraffouda** [comment]
so far issue is kinda fixed, except for the money.v [here](https://github.com/freeflowuniverse/crystallib/blob/development/crystallib/threefold/tfgrid_actions/web3gw/money.v) I didn't get what should be done there method names and args are not clear also some of them not compatible with the methods in rpc 
created a new issue here https://github.com/freeflowuniverse/crystallib/issues/285 to be tracked

**Comment by despiegk** [comment]
need to see together what we have and how far we got,
